bool ball_found
uint16 ball_center_x
uint16 ball_center_y
uint16 ball_radius